docs/conf.py

```diff
@@ -74,6 +74,10 @@
 # The reST default role (used for this markup: `text`) to use for all documents.
 #default_role = None
 
+# The Borg doc contain no or very little Python docs.
+# Thus, the primary domain is rst.
+primary_domain = 'rst'
```

Comment:
(This allows writing `.. class::`..., for example. Otherwise one would have to write `.. rst-class::`...)